docs/api.md

```diff
@@ -62,7 +62,7 @@ Some important differences between v1beta1/2 and v1beta3:
 * The namespace is required (for all namespaced resources) and has moved from a URL parameter to the path: `/api/v1beta3/namespaces/{namespace}/{resource_collection}/{resource_name}`. If you were not using a namespace before, use `default` here.
 * The names of all resource collections are now lower cased - instead of `replicationControllers`, use `replicationcontrollers`.
 * To watch for changes to a resource, open an HTTP or Websocket connection to the collection query and provide the `?watch=true` query parameter along with the desired `resourceVersion` parameter to watch from.
-* The `labels` query parameter has been renamed to `label-selector`.
+* The `labels` query parameter has been renamed to `labelSelector`.
```

Comment:
Yes. It was beta1->3 change: https://github.com/GoogleCloudPlatform/kubernetes/blob/de457c3e493234a819294bd6ec1f18101b9c4242/pkg/api/unversioned.go#L52